src/utils/socket.ts

```diff
@@ -28,7 +28,18 @@ export function connect(server) {
   })
   io.use(async (client, next) => {
     let userToken = client.handshake.headers['x-user-token']
-    const owner = await getOwnerFromToken(userToken)
+
```

Comment:
I'm not too sure what this socket file is doing since I'm not too familiar with it but I modified it to use the `x-transport-token`

Reply:
This is for the socket.io connection, that sends incoming messages to the app. Probably a good idea to set up a test for this (hooking up websocket, then receiving a message over it). 

You can see here how its done in JS in the electron app https://github.com/stakwork/sphinx-win-linux-desktop/blob/master/src/api/ws.ts#L30

Reply:
ok that makes sense so is this only used for the Mac, Win/linux clients or are there other clients using this code?

Reply:
All the clients attach the token like this. Not all using JS obviously